src/flp.rs

```diff
@@ -257,7 +257,7 @@ pub trait Type: Sized + Eq + Clone + Debug {
                     return Err(FlpError::Prove(format!(
                         "short prove randomness: got {}; want {}",
```

Comment:
Hmm, I guess what we want this error to convey is that the proof rand is too long? Maybe we something like: "short prove randomness: got {}; want at least {}"